### ISSUE TITLE
errors with debian 7

### ISSUE THREAD

**Issue by rmoriz** [hw-cookbooks/dpkg_autostart#9]
```
/usr/sbin/policy-rc.d: line 60: [: missing `]'
/usr/sbin/policy-rc.d: line 66: [: sd-agent: unary operator expected
/usr/sbin/policy-rc.d: line 73: [: start: unary operator expected
```

related to PRs:

https://github.com/hw-cookbooks/dpkg_autostart/pull/8
https://github.com/hw-cookbooks/dpkg_autostart/pull/7

**Comment by econnell** [comment]
#7 definitely fixes the problem on Ubuntu 14.04 as well.  The cookbook is pretty useless for me until that PR is merged

**Comment by chrisroberts** [comment]
merged + released
